examples/ensemble/plot_stacked_generalization_regression.py

```diff
@@ -0,0 +1,119 @@
+"""
+======================================================
+Stacked generalization applied to a regression problem
+======================================================
+
+This example shows how stacked generalization can be used to combine several
+regressors into a single stacked one that performs better than the best
+regressor.
+
+We use Boston's house pricing dataset to compare the mean squared error between
+three regressors (SVM, Lasso and Ridge regressions) and the combination of
+their outputs with a single linear regression. The following result is
+achieved.
```

Comment:
"The following result is achieved." seems not very informative.

Maybe add a conclusion, stating that the combined estimator achieve a better performance than each estimator alone.